docs/api.rst

```diff
@@ -345,16 +371,17 @@ tap
 
   tap :: (a -> *) -> Stream a -> Stream a
 
-Perform a side-effect for each event in stream.
+Perform a side effect for each event in a stream.
 
 .. code-block:: javascript
 
   stream:         -a-b-c-d->
   tap(f, stream): -a-b-c-d->
 
-For each event in stream, f is called, but the value of its result is ignored.
-If f fails (ie throws), then the returned stream will also fail. The stream
-returned by tap will contain the same events as the original stream.
+For each event in ``stream``, ``f`` is called, but the value of its result 
+is ignored. If ``f`` fails (i.e., throws an error), then the returned stream 
+will also fail. The stream returned by `tap` will contain the same events 
```

Comment:
`tap` needs double backticks, since this is RST. My markdown muscle memory still types single backticks all the time :/